### PR TITLE
Add text alignment icons

### LINKED ISSUE #646: [Request] Paragraph alignment icons
### Slack handle

`@arthur`

### What team and project is this for?

Email

### How will this icon be used?

Icons for text alignment: left, center & right.

### Major or minor?

Unsure

### Number of icons?

3

### PR DESCRIPTION
**Pull request by arthurgouveia**
Trying to add icons for text alignment purposes. I have no clue what I'm doing and would love to know about guidelines of adding new icons.

~Icons were fetched from https://github.com/Shopify/online-store-web/tree/master/app/components/RichTextEditor/components/EditImageModal/icons~

@nataliehercun synced with @AdamWhitcroft to design these new ones.

Closes #646.

### CODE REVIEW

File: packages/polaris-icons-raw/icons/polaris/text_alignment_center_major_monotone.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M5 4h10.005a1 1 0 0 1 0 2H5a1 1 0 0 1 0-2zm1 4a1.001 1.001 0 0 1 1.009-1h6.003c.548 0 .992.444.992 1 0 .552-.453 1-.998 1h-6.01A.999.999 0 0 1 6 8zm-1 2h10.005a1 1 0 0 1 0 2H5a1 1 0 0 1 0-2zm2 3h6.005a1 1 0 0 1 0 2H7a1 1 0 0 1 0-2z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text_alignment_center_major_monotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M5 4h10.005a1 1 0 0 1 0 2H5a1 1 0 0 1 0-2zm1 4a1.001 1.001 0 0 1 1.009-1h6.003c.548 0 .992.444.992 1 0 .552-.453 1-.998 1h-6.01A.999.999 0 0 1 6 8zm-1 2h10.005a1 1 0 0 1 0 2H5a1 1 0 0 1 0-2zm2 3h6.005a1 1 0 0 1 0 2H7a1 1 0 0 1 0-2z"/></svg>

```

File: packages/polaris-icons-raw/icons/polaris/text_alignment_left_major_monotone.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M4.994 4H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm-1 4a1 1 0 0 1 1.009-1h6.005c.548 0 .992.444.992 1 0 .552-.453 1-.998 1h-6.01a.997.997 0 0 1-.997-1zm1 2H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm0 3H11a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text_alignment_left_major_monotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M4.994 4H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm-1 4a1 1 0 0 1 1.009-1h6.005c.548 0 .992.444.992 1 0 .552-.453 1-.998 1h-6.01a.997.997 0 0 1-.997-1zm1 2H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm0 3H11a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2z"/></svg>

```

File: packages/polaris-icons-raw/icons/polaris/text_alignment_right_major.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+name: text alignment right
+set: major
+description: Icon indicating alignment of text on the right side
+keywords:
+  - text
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text_alignment_right_major.yml]: How about adding "rte" to the list of keywords for all of these icons?

File: packages/polaris-icons-raw/icons/polaris/text_alignment_right_major_monotone.svg [hunk 4]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M4.994 4H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm3 4a1 1 0 0 1 1.009-1h6.005c.548 0 .992.444.992 1 0 .552-.453 1-.998 1h-6.01a.997.997 0 0 1-.997-1zm-3 2H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm4 3H15a1 1 0 0 1 0 2H8.994a1 1 0 0 1 0-2z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text_alignment_right_major_monotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M4.994 4H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm3 4a1 1 0 0 1 1.009-1h6.005c.548 0 .992.444.992 1 0 .552-.453 1-.998 1h-6.01a.997.997 0 0 1-.997-1zm-3 2H15a1 1 0 0 1 0 2H4.994a1 1 0 0 1 0-2zm4 3H15a1 1 0 0 1 0 2H8.994a1 1 0 0 1 0-2z"/></svg>

```

**Comment by arthurgouveia** [comment]
@kaelig I've amended my commit: the `fill` to all elements and duplicated the `_monotone` to `_twotone` files, aside from updating changelog and re-exporting them.

**Comment by AdamWhitcroft** [comment]
I'm not sure where these icons came from. I'm going to ask around a bit, but they don't conform to our guidelines.

- Gaps between the horizontal bars are 1px, not the preferred minimum of 2px.
- The finished icon is 11px high, when it should be a round number for centering purposes.

We'd need to recreate these before shipping them.

**Comment by arthurgouveia** [comment]
@nataliehercun can you apply these guidelines to the SVGs I sent you over slack? I'll replace the icons here

**Comment by arthurgouveia** [comment]
@AdamWhitcroft I've pushed a commit with the new icons @nataliehercun designed/adjusted syncing with you.

File: packages/polaris-icons-raw/icons/polaris/text-alignment-center_major_monotone.svg [hunk 5]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM5 7L15 7C15.5522847 7 16 7.44771525 16 8 16 8.55228475 15.5522847 9 15 9L5 9C4.44771525 9 4 8.55228475 4 8 4 7.44771525 4.44771525 7 5 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM5 15L15 15C15.5522847 15 16 15.4477153 16 16 16 16.5522847 15.5522847 17 15 17L5 17C4.44771525 17 4 16.5522847 4 16 4 15.4477153 4.44771525 15 5 15z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-center_major_monotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M3 3h14a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm2 4h10a1 1 0 0 1 0 2H5a1 1 0 1 1 0-2zm-2 4h14a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2zm2 4h10a1 1 0 0 1 0 2H5a1 1 0 0 1 0-2z"/></svg>
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-center_major_twotone.svg [hunk 6]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM5 7L15 7C15.5522847 7 16 7.44771525 16 8 16 8.55228475 15.5522847 9 15 9L5 9C4.44771525 9 4 8.55228475 4 8 4 7.44771525 4.44771525 7 5 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM5 15L15 15C15.5522847 15 16 15.4477153 16 16 16 16.5522847 15.5522847 17 15 17L5 17C4.44771525 17 4 16.5522847 4 16 4 15.4477153 4.44771525 15 5 15z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-center_major_twotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M3 3h14a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm2 4h10a1 1 0 0 1 0 2H5a1 1 0 1 1 0-2zm-2 4h14a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2zm2 4h10a1 1 0 0 1 0 2H5a1 1 0 0 1 0-2z"/></svg>
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_monotone.svg [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
+ <path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM3 7L13 7C13.5522847 7 14 7.44771525 14 8 14 8.55228475 13.5522847 9 13 9L3 9C2.44771525 9 2 8.55228475 2 8 2 7.44771525 2.44771525 7 3 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM3 15L13 15C13.5522847 15 14 15.4477153 14 16 14 16.5522847 13.5522847 17 13 17L3 17C2.44771525 17 2 16.5522847 2 16 2 15.4477153 2.44771525 15 3 15z"/>
+</svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_monotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M3 3h14a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm0 4h10a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm0 4h14a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2zm0 4h10a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2z"/></svg>
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_monotone.svg [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
+ <path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM3 7L13 7C13.5522847 7 14 7.44771525 14 8 14 8.55228475 13.5522847 9 13 9L3 9C2.44771525 9 2 8.55228475 2 8 2 7.44771525 2.44771525 7 3 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM3 15L13 15C13.5522847 15 14 15.4477153 14 16 14 16.5522847 13.5522847 17 13 17L3 17C2.44771525 17 2 16.5522847 2 16 2 15.4477153 2.44771525 15 3 15z"/>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_monotone.svg]:
```suggestion
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_monotone.svg [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_monotone.svg]:
```suggestion
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_twotone.svg [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20">
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_twotone.svg]:
```suggestion
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_twotone.svg [hunk 11]
```diff
@@ -0,0 +1,3 @@
...
+ <path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM3 7L13 7C13.5522847 7 14 7.44771525 14 8 14 8.55228475 13.5522847 9 13 9L3 9C2.44771525 9 2 8.55228475 2 8 2 7.44771525 2.44771525 7 3 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM3 15L13 15C13.5522847 15 14 15.4477153 14 16 14 16.5522847 13.5522847 17 13 17L3 17C2.44771525 17 2 16.5522847 2 16 2 15.4477153 2.44771525 15 3 15z"/>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_twotone.svg]:
```suggestion
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_twotone.svg [hunk 12]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-left_major_twotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M3 3h14a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm0 4h10a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm0 4h14a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2zm0 4h10a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2z"/></svg>
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-right_major_monotone.svg [hunk 13]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM7 7L17 7C17.5522847 7 18 7.44771525 18 8 18 8.55228475 17.5522847 9 17 9L7 9C6.44771525 9 6 8.55228475 6 8 6 7.44771525 6.44771525 7 7 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM7 15L17 15C17.5522847 15 18 15.4477153 18 16 18 16.5522847 17.5522847 17 17 17L7 17C6.44771525 17 6 16.5522847 6 16 6 15.4477153 6.44771525 15 7 15z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-right_major_monotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M3 3h14a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm4 4h10a1 1 0 0 1 0 2H7a1 1 0 1 1 0-2zm-4 4h14a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2zm4 4h10a1 1 0 0 1 0 2H7a1 1 0 0 1 0-2z"/></svg>
```

File: packages/polaris-icons-raw/icons/polaris/text-alignment-right_major_twotone.svg [hunk 14]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" viewBox="0 0 20 20"><path fill="#637381" d="M3 3L17 3C17.5522847 3 18 3.44771525 18 4 18 4.55228475 17.5522847 5 17 5L3 5C2.44771525 5 2 4.55228475 2 4 2 3.44771525 2.44771525 3 3 3zM7 7L17 7C17.5522847 7 18 7.44771525 18 8 18 8.55228475 17.5522847 9 17 9L7 9C6.44771525 9 6 8.55228475 6 8 6 7.44771525 6.44771525 7 7 7zM3 11L17 11C17.5522847 11 18 11.4477153 18 12 18 12.5522847 17.5522847 13 17 13L3 13C2.44771525 13 2 12.5522847 2 12 2 11.4477153 2.44771525 11 3 11zM7 15L17 15C17.5522847 15 18 15.4477153 18 16 18 16.5522847 17.5522847 17 17 17L7 17C6.44771525 17 6 16.5522847 6 16 6 15.4477153 6.44771525 15 7 15z"/></svg>
```
Comment (kaelig) [inline packages/polaris-icons-raw/icons/polaris/text-alignment-right_major_twotone.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path fill="#637381" d="M3 3h14a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2zm4 4h10a1 1 0 0 1 0 2H7a1 1 0 1 1 0-2zm-4 4h14a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2zm4 4h10a1 1 0 0 1 0 2H7a1 1 0 0 1 0-2z"/></svg>
```

**Comment by arthurgouveia** [comment]
@kaelig thanks for that! I'll amend my commit.

Out of curiosity, what are the params you're using for SVGO? I've asked the designer to follow these guidelines on the vault: https://vault.shopify.io/pages/1756-Exporting-icons-SVG

File: packages/polaris-icons/CHANGELOG.md [hunk 15]
```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 
 ## Unreleased
 
+## 3.4.1 - 2019-08-07
```
Comment (kaelig) [inline packages/polaris-icons/CHANGELOG.md]:
```suggestion
```

**Comment by kaelig** [comment]
The vault instructions are a great baseline but I've done a lot of SVG optimization in my days... And there's nothing that beats SVGOMG with a manual visual check.

**Pull request merged**